modules/virt-adding-a-boot-source-web.adoc

```diff
@@ -37,7 +37,7 @@ You do not need special privileges to create a VM from a template with an operat
 +
 [NOTE]
 ====
-Provided boot sources are updated automatically to the latest version of the operating system. For auto-updated boot sources, persistent volume claims (PVCs) are created with the cluster's default storage class. If you select a different default storage class after configuration, you must delete the existing data volumes in the cluster namespace that are configured with the previous default storage class.
+Provided boot sources are updated automatically to the latest version of the operating system. For auto-updated boot sources, persistent volume claims (PVCs) or volume snapshots are created with the cluster's default storage class. If you select a different default storage class after configuration, you must delete the existing boot sources in the cluster namespace that are configured with the previous default storage class.
```

Comment:
persistent volume claims (PVCs) *or* volume snapshots here but in the previous note you use *and*.  Just pointing it out in case you'd like to be consistent.